frontend/src/pages/Login.jsx

```diff
@@ -1,11 +1,39 @@
-import './stylesheets/Login.css'
+import { useState, useEffect } from 'react';
+import { login } from './util/login'
+
 import Header from '../components/Header'
 import Footer from '../components/Footer'
 
+import './stylesheets/Login.css'
+
+
 function Login() {
```

Comment:
I'd rename this to be `LoginComponent`, compared to `Login` as it may be confused for a util function (see the way your util function has the same name?)

Reply:
Renamed to `LoginPage`!